packages/augur-ui/src/modules/modal/add-funds.tsx

```diff
@@ -213,27 +200,26 @@ export const AddFunds = ({
                   defaultSelection={SWAP_ID}
                 />
               </div>
-              {poolSelected && (
-                <Swap
-                  address={loginAccount.address}
-                  balances={loginAccount.balances}
-                  toToken={tokenToAdd}
-                  fromToken={initialSwapToken ? initialSwapToken : null}
-                  ETH_RATE={ETH_RATE}
-                  REP_RATE={REP_RATE}
-                  config={config}
-                  useSigner={useSigner}
-                />
-              )}
+
+              <Swap
```

Comment:
Anything that is passed to this component but isn't used on otherwise should just be refactored into this component directly. we don't want to pass props unless they are actually needed to be passed like a marketID or something.